README.md

```diff
@@ -33,8 +33,10 @@ None.
 
 ## Role Variables ##
 
-- `jetbrainsmono_version` - the version of the JetBrainsMono font to
-  download from JetBrains/JetBrainsMono.  Defaults to "2.241".
+| Variable | Description | Default | Required |
+|----------|-------------|---------|----------|
+| jetbrainsmono_version | The version of the JetBrainsMono font to
+  download from JetBrains/JetBrainsMono. | `2.241` | No |
```

Comment:
Breaking the line causes a failure in rendering:

![image](https://user-images.githubusercontent.com/50747025/130510135-af426186-dfb8-441a-8926-1ecc51ff19ac.png)


```suggestion
| jetbrainsmono_version | The version of the JetBrainsMono font to download from JetBrains/JetBrainsMono. | `2.241` | No |
```

Reply:
Good catch!